docs/source/tutorial/basic.rst

```diff
@@ -314,24 +326,43 @@ In this case, :meth:`~Link.zerograds` is automatically called by the update meth
    ...     return loss
    >>> optimizer.update(lossfun, args...)
 
```

Comment:
It is more kind to write something like "see optimizer.update for full specification".